capstone/src/main/webapp/config.js

```diff
@@ -0,0 +1,5 @@
+var config = {
```

Comment:
Should this be checked in?

Reply:
no, this was fixed (will be fixed) in a later PR... our gitignores messed up in a merge.